src/components/DirectAnswer.tsx

```diff
@@ -71,14 +74,11 @@ export function DirectAnswer(props: DirectAnswerProps): JSX.Element | null {
   }
 
   const cssClasses = getCssClassesForAnswerType(composedCssClasses, directAnswerResult.type);
-  const title = directAnswerResult.type === DirectAnswerType.FeaturedSnippet
-    ? directAnswerResult.value as string // TODO: update with other direct answer changes
-    : `${directAnswerResult.entityName} / ${directAnswerResult.fieldName}`;
-  const description: ReactNode = directAnswerResult.type === DirectAnswerType.FeaturedSnippet
-    ? renderHighlightedValue(directAnswerResult.snippet, { highlighted: cssClasses.highlighted })
-    : directAnswerResult.value as string; // TODO: update with other direct answer changes
-  const link = directAnswerResult.relatedResult.link;
+  const containerCssClasses = classNames(cssClasses.directAnswerContainer, {
```

Comment:
I think we want to use `tw-merge` instead of `classNames` now, right?

Reply:
sure, we should probably update other areas that uses classNames too. But that's gonna be a separate item

Reply:
sounds good!